dtfs-central-api/api-tests/v1/tfm/amendments/tfm-amendments-get.api-test.js

```diff
@@ -66,13 +70,6 @@ describe('GET TFM amendments', () => {
           dealId: expect.any(String),
           facilityId: expect.any(String),
           version: 1,
-          auditRecord: {
```

Comment:
I'm probably being dumb, why is this no longer needed? Are we doing a partial match and ignoring this or am I missing something?

Reply:
This (as you can see) confused me too - amendments isn't its own collection, its just a property within `tfm-facilities`. So when the endpoint returns an amendment, it wont return the facility-level `auditRecord` with it.
Do you think I should add an additional test to get the facility and check it's auditRecord has been changed?

Reply:
This (as you can see) confused me too - amendments isn't its own collection, its just a property within `tfm-facilities`. So when the endpoint returns an amendment, it wont return the facility-level `auditRecord` with it.
Do you think I should add an additional test to get the facility and check it's auditRecord has been changed?

Reply:
Ooh that makes a lot of sense, yeah we should add just one that the base level record as an auditRecord that is updated on it :)